onboarding/geo/README.md

```diff
@@ -0,0 +1,334 @@
+# Geospatial Data Science Onboarding Guide
+
+Welcome! In this onboarding guide, we will introduce you to the fascinating field of geospatial data analysis and set the stage for your learning journey.
```

Comment:
I feel a bit like chat gpt is talking ;). Maybe also try https://huggingface.co/chat/ to help you along your journey.

What could help is to add a table of contents with links. Often, what I also would do in the intro is to discuss what someone will gain after reading the whole document. Then, per section, I would again reiterate what the take-aways are of that particular section. You cover that well in "What to expect", maybe you can add a bit in the intro already.

One of the most famous computer science professors of all times, Donald Knuth, has written a good article on tips for mathematical writing, https://jmlr.csail.mit.edu/reviewing-papers/knuth_mathematical_writing.pdf . We could of course argue that an onboarding guide is not necessarily the same as a mathematical article, I tend to be more of the school of thoughts that there have to be some good learnings there :).